mergify_engine/context.py

```diff
@@ -1012,6 +1013,16 @@ async def _get_consolidated_data(self, name: str) -> ContextAttributeType:
                 return -1
             return position
 
+        elif name == "queue-speculative-check-elapsed-time":
```

Comment:
that does not work for inplace?

Reply:
Yes, it does :)

Reply:
The name sounds strange then?

Reply:
When even we rebase a PR instead of creating a draft one, it's still a speculative checking.

Reply:
Err no, in place cannot be speculative. Speculative means we _speculate_ the first one work, so the check of the first PR in the queue is *not* speculative.

Reply:
Ok, I see your point! so maybe just `queue-checks-elapsed-time`.